### PR TITLE
Add Poloniex and Bitfinex

### PR DESCRIPTION
**Pull request by rajivshah3**

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ websites:
       tfa: Yes
       software: Yes
 
+ - name: Bitfinex
+      url: https://www.bitfinex.com
+      img: bitfinex.png
+      tfa: Yes
+      software: Yes
+      doc: https://support.bitfinex.com/hc/en-us/sections/203192029-Security
+
```
Comment (mxxcon) [inline _data/cryptocurrencies.yml]: Please correct indentations here. In YAML format spacing matters.

**Review by mxxcon — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 1 above)
Comment (rajivshah3) [reply _data/cryptocurrencies.yml]: Done

Comment (mxxcon) [reply _data/cryptocurrencies.yml]: I noticed that their docs page states that they also support SMS? If that's the case, perhaps add it.

Comment (rajivshah3) [reply _data/cryptocurrencies.yml]: My bad, fixed that

**Review by mxxcon — APPROVED** [review]

**Comment by mxxcon** [comment]
In the future please submit one site per pull request so that they can be reviewed independently of each other.

**Comment by rajivshah3** [comment]
I originally intended to, but when I submitted the commit for bitfinex it added on to the existing one for Poloniex

**Comment by mxxcon** [comment]
To get around such situations, before you make your additions, in your own repo create separate branches for each site and make changes there. Then you can create pull requests from those separate branches.

**Review by fpigerre — APPROVED** [review]

**Comment by fpigerre** [comment]
Thanks for the pull request @rajivshah3! 👍

**Comment by Carlgo11** [comment]
How did not travis fail here?
Got any ideas @stephengroat?

**Comment by stephengroat** [comment]
travis implemented a change that silently broke system using implicit matrix expansion https://github.com/travis-ci/travis-ci/issues/4681

**Pull request merged**